### ISSUE TITLE
sudo fatrat command not found

### ISSUE THREAD

**Issue by koopapa62** [screetsec/TheFatRat#737]
Hello here is my issue :

sudo fatrat
sudo: fatrat: command not found

please help me i'm stuck on this for an hour

**Comment by koopapa62** [comment]
update : i checked #244  and i did all successfully  but unfortunately when I run "apt-get update -y && apt-get install mingw32 && apt-get install mingw-w64 -y" it says 
![image](https://user-images.githubusercontent.com/86924493/184504247-1914a425-0c35-48b1-802c-8db09fca5d74.png)

and that's annoying asf can't find a solution anywhere

thank you

**Comment by Morsmalleo** [comment]
What's your Kali version???

**Comment by koopapa62** [comment]
this issue is resolved thank you tho, but i got another problem that you could help me with, everytime I open a meterpreter session, i run the payload on my main computer, nothing pops up, no session get created and I think that's actually an issue with my IP adress, my local IP adress on my vbox is not the same as the one on my computer even if I enabled bridged connexion, it's been some weeks I have this problem and that's sooo annoying, how could i do to get a meterpreter session working on any device ? and not only on my computer if you understand what I mean

thank you

**Comment by Morsmalleo** [comment]
> this issue is resolved thank you tho, but i got another problem that you could help me with, everytime I open a meterpreter session, i run the payload on my main computer, nothing pops up, no session get created and I think that's actually an issue with my IP adress, my local IP adress on my vbox is not the same as the one on my computer even if I enabled bridged connexion, it's been some weeks I have this problem and that's sooo annoying, how could i do to get a meterpreter session working on any device ? and not only on my computer if you understand what I mean
> 
> thank you

Most likely because of your IP or Port, what IP type does your IP use, is it `eth` or `wlan` you can check this by running `ifconfig` in the Kali Terminal, you should be greeted with your IP address next to something like `wlan0` or `eth0`

**Comment by koopapa62** [comment]
yeah I use the eth0, so I succeeded to get the same ip on vbox, but still a problem, i got no internet on my vbox, I just put the same MAC adress as my computer so it has the same ip but now i got no internet on vbox, i guess that's because vbox is unable to use wi-fi internet directly

**Comment by Morsmalleo** [comment]
> yeah I use the eth0, so I succeeded to get the same ip on vbox, but still a problem, i got no internet on my vbox, I just put the same MAC adress as my computer so it has the same ip but now i got no internet on vbox, i guess that's because vbox is unable to use wi-fi internet directly

You could have selected the wrong option in `Bridged Adapters` because your VM should be emulating your wifi card as an ethernet connection, so therefor any wifi you're connected to should work for the VM, hence the `eth0`, Mac address won't do much for you here

**Comment by koopapa62** [comment]
And how could I fix that ? thank you for the support btw I have several problems with internet and Ips since I installed kali

**Comment by koopapa62** [comment]
Because the problem is that if I don't set the MAC adress, even bridging the connexion, my IP won't be the same as my computer, so I can't test the payloads in my meterpreter

**Comment by koopapa62** [comment]
![image](https://user-images.githubusercontent.com/86924493/184534487-8c88182d-a32b-42f0-813a-494cadc71c06.png)

![image](https://user-images.githubusercontent.com/86924493/184534501-4182457b-7571-403f-bec9-2d3502b9490d.png)

![image](https://user-images.githubusercontent.com/86924493/184534517-e2cf00c0-0f75-469e-98c1-a2ee847b22da.png)


as you can see even bridged, if I don't put the same mac adress the ip is not corresponding

**Comment by peterpt** [comment]
it is sudo command that is not found or fatrat ?
for sudo = apt install sudo

For fatrat you have to install it sucefully for fatrat to work , any issue while installing fatrat will not enable fatrat script . fatrat will only work if everything is installed correctly during setup .
